motion_tracker/utils/crop_vis.py

```diff
@@ -51,3 +52,32 @@ def show_img(img, box_coords=None, window_name="Happy Dance Image",
     cv2.waitKey(msec_to_show_for)
     cv2.destroyWindow(window_name)
 
+def show_img_from_filepath(img_filename, imagenet=True): 
+    """Show an image and it's bounding box from a given filepath
+
+    Args: 
+    ----
+        img_filename: str
+        imagenet: boolean 
+            Holds whether it's an imagenet or alov image. 
+    """
+
+    raw_img_dir = 'data/imagenet/images/' if imagenet else 'work/alov/images/'
+    img_metadata_fp = 'work/imagenet/parsed_bb2.csv' if imagenet else 'work/alov/parsed_bb.csv'
+
+    img_metadata_df = pd.read_csv(img_metadata_fp)
+    if imagenet: 
+        img_metadata = img_metadata_df.query("filename == @img_filename")
+        box_coords = Coords(img_metadata.x0, img_metadata.y0, 
+                            img_metadata.x1, img_metadata.y1)
+    else: 
+        img_metadata = img_metadata_df.query("filename_start == @img_filename")
+        box_coords = Coords(img_metadata.x0_start, img_metadata.y0_start, 
+                            img_metadata.x1_start, img_metadata.y1_start)
+
+    img = cv2.imread(raw_img_dir + img_filename)
+    show_img(img, box_coords.as_array())
+
```

Comment:
We don't need to keep this, but it might be nice to keep around for a little while. Basically, you just put in the image filename, and what source it came from, and with the way we have everything structured right now, it displays that image with it's original bounding box. It was kind of useful for debugging, but might have limited use in the future (but it could also be useful to show results or pull select images with their original bounding boxes).